zrml/orderbook/README.md

```diff
@@ -1,3 +1,46 @@
-# Orderbook Module
+# Orderbook Pallet
 
-A module to trade shares using a naive on-chain orderbook.
+A pallet of an on-chain order book, which allows to exchange the market's base
+asset for outcome assets and vice versa.
+
+## Overview
+
+The order book can be set as a market's scoring rule. It allows to place,
+partially or fully fill and remove orders.
+
+## Terminology
+
+- `maker_partial_fill`: The partial amount of what the maker wants to fill.
```

Comment:
How can a maker fill an order? Wouldn't that make them a taker?

Reply:
Haha, good point @Robiquet .. Malte already pointed this out extensively. Good that you found another instance for this.

Reply:
a0e0f21bb21a7e1044cfaa4ae95d23329a30a394